packages/wallets/src/components/Base/ModalStepWrapper/ModalStepWrapper.tsx

```diff
@@ -43,7 +44,11 @@ const ModalStepWrapper: FC<PropsWithChildren<TModalStepWrapperProps>> = ({
 
     const Footer = () =>
         hasRenderFooter ? (
-            <div className='wallets-modal-step-wrapper__footer' data-testid='dt_modal_step_wrapper_footer'>
+            <div
+                className='wallets-modal-step-wrapper__footer'
+                data-testid='dt_modal_step_wrapper_footer'
+                style={{ display: shouldHideFooter ? 'none' : 'inlineBlock' }}
```

Comment:
should this be moved into the scss file instead?

Reply:
Done